apps/design-land/src/app/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,14 @@
+<ul>
```

Comment:
Should be `daff-button-set` and `daff-button`s.

Reply:
<img width="542" alt="Screen Shot 2019-09-03 at 3 29 04 PM" src="https://user-images.githubusercontent.com/9029654/64202746-dc9cf680-ce5f-11e9-9eab-f0efe0e39061.png">

It's not much better.

Reply:
addressed in b827ea8b